test/FunctionalTests/Tests/DataServices/UnitTests/DesignT4UnitTests/ODataT4CodeGeneratorTestDescriptors.cs

```diff
@@ -46,6 +49,24 @@ public class ODataT4CodeGeneratorTestsDescriptor
             public Action<string, bool, bool> Verify { get; set; }
         }
 
+        internal static void ValidateXMLFile(string tempFilePath)
+        {
+            XmlDocument doc = new XmlDocument();
+            doc.Load(tempFilePath);
+        }
+
+        internal static void ValidateEdmx(string tempFilePath)
+        {
+            string edmx = File.ReadAllText(tempFilePath);
+            var stringReader = new StringReader(edmx);
+            var xmlReader = XmlReader.Create(stringReader);

```

Comment:
It's good practice to wrap the readers in a using statement to ensure that the object is properly closed/disposed and resources freed
`using (XmlReader reader = XmlReader.Create(inputUrl)) {...} // same for stringreader`

Reply:
fixed